src/main.ts

```diff
@@ -1,14 +1,221 @@
+import { DatasetFlowDetailsModule } from "./app/dataset-flow/dataset-flow-details/dataset-flow-details.module";
+import { DatasetCreateModule } from "./app/dataset-create/dataset-create.module";
+
+import { ModalModule } from "./app/common/components/modal/modal.module";
+import { MarkdownModule } from "ngx-markdown";
+import { ClipboardModule } from "@angular/cdk/clipboard";
+import { provideAnimations } from "@angular/platform-browser/animations";
+import { BrowserModule, bootstrapApplication } from "@angular/platform-browser";
+import { CommonModule } from "@angular/common";
+import { MAT_RIPPLE_GLOBAL_OPTIONS } from "@angular/material/core";
+import { logError } from "./app/common/helpers/app.helpers";
+import { firstValueFrom } from "rxjs";
+import { HIGHLIGHT_OPTIONS } from "ngx-highlightjs";
+import { apolloCache } from "./app/common/helpers/apollo-cache.helper";
+import { ErrorTexts } from "./app/common/values/errors.text";
+import { ToastrService, ToastrModule } from "ngx-toastr";
+import { onError } from "@apollo/client/link/error";
+import { ApolloLink, Operation, NextLink } from "@apollo/client/core";
+import { LocalStorageService } from "./app/services/local-storage.service";
+import { AppConfigService } from "./app/app-config.service";
+import { ErrorHandlerService } from "./app/services/error-handler.service";
+import { SpinnerService } from "./app/common/components/spinner/spinner.service";
+import { SpinnerInterceptor } from "./app/common/components/spinner/spinner.interceptor";
+import {
+    HTTP_INTERCEPTORS,
+    HttpHeaders,
+    withInterceptorsFromDi,
+    provideHttpClient,
+    HttpClient,
+} from "@angular/common/http";
+import { SearchService } from "./app/search/search.service";
+import { SearchApi } from "./app/api/search.api";
+import { NavigationService } from "./app/services/navigation.service";
+import { LoggedUserService } from "./app/auth/logged-user.service";
+import { LoginService } from "./app/auth/login/login.service";
+import { HttpLink } from "apollo-angular/http";
+import { DatasetSubscriptionsService } from "./app/dataset-view/dataset.subscriptions.service";
+import { DatasetService } from "./app/dataset-view/dataset.service";
+import { DatasetPermissionsService } from "./app/dataset-view/dataset.permissions.service";
+import { DatasetApi } from "./app/api/dataset.api";
+import { AuthApi } from "./app/api/auth.api";
+import { Apollo, APOLLO_OPTIONS } from "apollo-angular";
+import AppValues from "./app/common/values/app.values";
+import { provideRouter, withComponentInputBinding, withRouterConfig } from "@angular/router";
+import { routes } from "./app/app-routing";
+import { NgxGraphModule } from "@swimlane/ngx-graph";
+
+const Services = [
+    Apollo,
+    AuthApi,
+    DatasetApi,
+    DatasetPermissionsService,
+    DatasetService,
+    DatasetSubscriptionsService,
+    HttpLink,
+    LoginService,
+    LoggedUserService,
+    NavigationService,
+    SearchApi,
+    SearchService,
```

Comment:
Are you sure we still have to list them explicitly? I thought they are auto-linked normally from components.

Reply:
Left only  `Apollo`